src/css/structure/modules/infragistics.ui.scroll.css

```diff
@@ -93,6 +100,20 @@
 }
 
 /* Arrow Down */
+.igscroll-downarrow {
+	bottom: 15px;
+	position: absolute;
+}
+
+.igscroll-downarrow-singlescrollbar {
+	bottom: 0px !important;
```

Comment:
I have some more commits that address this and the native scrollbars as well, so this shouldn't be merged yet.